components/admin_console/permission_schemes_settings/permission_schemes_settings.jsx

```diff
@@ -3,18 +3,21 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import {FormattedMessage} from 'react-intl';
+import {FormattedMessage, FormattedHTMLMessage} from 'react-intl';
```

Comment:
Please can you use `FormattedMessage` with parameters containing React elements instead of `FormattedHTMLMessage`? We're trying to (gradually) remove `FormattedHTMLMessage` from the codebase. (Actually it occurs to me this should probably be added to the dev decisions list, given we made the decision a couple of years ago).